<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAddressDialog</class>
 <widget class="QDialog" name="EditAddressDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Address</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleWidget, #bodyWidget { background-color: white; }

QLineEdit { border: none; border-bottom: 2px solid #3cb0e5; padding: 3px 5px; background: transparent; } QLineEdit:disabled { border-color: #899399; }</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="editAddressDialogLayout" stretch="0,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QVBoxLayout" name="titleWidgetLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#titleLabel { text-transform: uppercase; font-family: 'Montserrat'; font-size: 14px; font-weight: bold; }</string>
        </property>
        <property name="text">
         <string>TITLE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bodyWidget" native="true">
     <layout class="QVBoxLayout" name="bodyWidgetLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="inputWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="inputWidgetLayout" stretch="1,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="labelLabel">
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>LABEL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="labelEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans</family>
              </font>
             </property>
             <property name="toolTip">
              <string>The label associated with this address book entry</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="addressLabel">
             <property name="font">
              <font>
               <family>Montserrat</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ADDRESS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="addressEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Open Sans</family>
              </font>
             </property>
             <property name="toolTip">
              <string>The address associated with this address book entry. This can only be modified for sending addresses.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="buttonWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#buttonWidget { background-color: #f3f4f7; }

QPushButton { font-family: 'Montserrat'; text-transform: uppercase; font-size: 10px; border: 1px solid gray; padding: 0px 15px; border-radius: 13px; min-height: 26px; max-height: 26px; background-color: white; }

QPushButton:hover, QPushButton:pressed { background-color: #1c99d2; color: white; }
QPushButton:disabled { background-color: white; color: gray; }</string>
        </property>
        <layout class="QHBoxLayout" name="buttonWidgetLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="font">
            <font>
             <family>Montserrat</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="confirmButton">
           <property name="font">
            <font>
             <family>Montserrat</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#confirmButton { background-color: #3cb0e5; color: white; border: 1px solid gray; }
#confirmButton:hover, #confirmButton:pressed { background-color: #1c99d2; }</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>labelEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>confirmButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
